### PR TITLE
imp(docs): Add readme

### PR DESCRIPTION
**Pull request by MalteHerrmann**
This PR adds the readme for the evmOS repository.

## Summary by CodeRabbit

- **New Features**
	- Introduced a comprehensive README that details evmOS features, benefits, and integration processes.
	- Added a roadmap outlining future developments and partnership opportunities for evmOS.
- **Improvements**
	- Enhanced changelog to include a reference to the new README, indicating ongoing improvements in documentation and user guidance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,123 @@
 # evmOS
+
```
Comment (MalteHerrmann) [inline README.md:2]: remove
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1 +1,121 @@
...
+Becoming an evmOS partner, you will get:
+
+- Full access to evmOS modules and updates
+- Smooth onboarding experience for an existing or new Cosmos chain
+- Full access to product partnerships (block explorers, RPCs, indexers etc.)
+- Continuous upgrades, access to product and engineering support
+
+evmOS is production ready, and suitable for usage in mission-critical environments such as application-specific EVMs.
+
+## Plug-in evmOS into your chain
```
Comment (0xstepit) [inline README.md:16]: Please be consistent with American or European title style.

File: README.md [hunk 3]
```diff
@@ -1 +1,121 @@
...
+
+### Configurations
+
+Our solution is engineered to provide unique flexibility, empowering you to tailor every aspect of your Ethereum Virtual Machine (EVM) environment. Whether you're launching a new blockchain or optimizing an existing one, evmOS offers a suite of features designed to meet the unique demands of your project.
+
+#### Powerful Defaults
+
+evmOS’ modules come out of the box with defaults that will get you up and running instantly. 
+
+When integrating all available modules you will get a *permissionless EVM-enabled* blockchain that *exposes JSON-RPC* endpoints for connectivity with all EVM tooling like wallets ([MetaMask](https://metamask.io/), [Rabby](rabby.io), and others) or block explorers ([Blockscout](https://docs.blockscout.com/) and others). You will have access to *all of evmOS’ EVM extensions*, which enable access to chain-native functionality through Solidity smart contracts.
```
Comment (0xstepit) [inline README.md:30]: Maybe we can add the hyperlink to Solidity too

File: README.md [hunk 4]
```diff
@@ -1 +1,121 @@
...
+Ethereum-compatible means,
+that the EVM can be set up to run every transaction
+that is valid on Ethereum while the handling of the transactions
+can diverge in e.g. result or cost.
+
+We like to coin the term **forward-compatible**
+as a description of our EVM solution,
+meaning that any evmOS chain can run any valid smart contract
+from Ethereum but is able to implement new features that are
+not (yet) available on the standard Ethereum EVM,
```
Comment (0xstepit) [inline README.md:85]: Ethereum is already in the EVM name, seems redundant to read 😅
```suggestion
not (yet) available on the standard Ethereum EVM,
```

**Review by 0xstepit — APPROVED** [review]
Very cool description @MalteHerrmann! Could you just please lint the markdown to have a consistent style? Like max columns in a row, blank line between UL or OL?

**Pull request merged**